### ISSUE TITLE
Users need to understand why they're not given a full permit

### ISSUE THREAD

**Issue by EUzkuraityte** [alphagov/verify-local-patterns#591]
Evidence form latest round of user research on Parking Permits (case when LA receives an unverified Address and chooses to issue the user with a 3 month permit)
![screen shot 2017-06-16 at 13 48 01](https://user-images.githubusercontent.com/20285115/27227236-8e8af7f0-529a-11e7-9b54-2a99cd3c29be.png)

**Comment by lizziebruce** [comment]
I'll do some thinking on new content for this when it comes up in the priority in the ready to do list. I think this is a duplicate of another issue re current address unverified but I can't find that issue so can't close this one or reference the other one.

**Comment by lizziebruce** [comment]
Content fix

(See above for before)

After:

![screen shot 2017-06-23 at 11 40 35](https://user-images.githubusercontent.com/27814324/27478801-1df2a6f4-5809-11e7-8ffb-c7d1bfa61e58.png)

After – hidden text showing:

![screen shot 2017-06-23 at 11 43 33](https://user-images.githubusercontent.com/27814324/27478830-3da46c30-5809-11e7-857e-4caea4110263.png)

**Comment by suryanaga** [comment]
@lizziebruce I'm not sure that hiding the addresses in the box will change user's dissatisfaction about this. I feel like this issue is much more around setting expectations at the beginning of the process.

**Comment by lizziebruce** [comment]
@sanjaypoyzer this content fix is addressing the cause of the dissatisfaction as outlined on the UR card attached to this issue, which says users thought the council was saying they were ineligible rather than GOV.UK Verify. So the new content references GOV.UK Verify a lot more. We could also add 'by GOV.UK Verify' to the end of the page title. 

The UR card says that specifically "those interpreting it in this way" i.e. that the council wasn't letting them have a permit because they'd not lived there long enough, were dissatisfied, so I think this is the right place to address it. 
It might also be worth flagging on the prove identity page that you'll usually need to have lived somewhere 3 months for Verify to work though.

**Comment by suryanaga** [comment]
Yeah, I think the prove identity page is where to flag it, although it's more about saying that we'll (probably) give you a 3 month permit if you've just moved in.

**Comment by lizziebruce** [comment]
Content fix to flag up possibility of 3 month permit at choose Verify stage - but we might be changing this page for #588 

![screen shot 2017-06-27 at 14 52 53](https://user-images.githubusercontent.com/27814324/27591177-d2080a02-5b48-11e7-8f05-60d2a3a625f7.png)

**Comment by EUzkuraityte** [comment]
Additional evidence from Sunderland re: user dissatisfaction for getting a 3 month permit only

![image](https://user-images.githubusercontent.com/20285115/27861082-345b6ca6-6177-11e7-91eb-028d7e3e0736.png)

**Comment by lizziebruce** [comment]
@EUzkuraityte @sanjaypoyzer don't think there's a lot we can do about this. It's the actuality that they are objecting to not, the user journey. SOme councils won't even be giving them a 3 month permit.

**Comment by suryanaga** [comment]
Well the user need stated on this issue is that people need to understand _why_. How happy they are about it doesn't really tell us much about user needs.

**Comment by ingasiler** [comment]
Findings from UR lab day on 11 July

![screen shot 2017-07-17 at 18 15 12](https://user-images.githubusercontent.com/19390851/28280526-e769b35e-6b1b-11e7-95c6-23eef8e86cb6.png)

**Comment by lizziebruce** [comment]
Issue is that last blue screen of the Hub, after IDP and before service, says your **identity** has been verified (but this doesn't necessarily mean their address has). 

Copy change suggestion for service page:

"Your identity has been verified, but your new address is not yet verified. This means you can get a 12 month..."

**Comment by lizziebruce** [comment]
Have applied this:

![screen shot 2017-07-21 at 14 47 19](https://user-images.githubusercontent.com/27814324/28466235-b0f1b6ea-6e23-11e7-988d-d08ed2c8e201.png)
